Library/Formula/dcd.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
+
+  depends_on "dmd" => :build
+
+  def install
+    system "make"
+    bin.install "bin/dcd-client", "bin/dcd-server"
```

Comment:
Alright.

Garg, okay. Yeah in that case we'll need to be as explicit as we have been here. We could try and be fancier, but simple works in this case.